project.clj

```diff
@@ -1,6 +1,6 @@
 (def jruby-version "9.2.11.1")
 
-(defproject puppetlabs/jruby-deps "9.2.11.0-2-SNAPSHOT"
+(defproject puppetlabs/jruby-deps "9.2.11.1-1-SNAPSHOT"
```

Comment:
we start with `-1` and not `-0`?

Reply:
That's what I'm seeing for all the previous releases.

Reply:
ookay!